pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -58,16 +58,34 @@ func (a *Operator) syncOperatorGroups(obj interface{}) error {
 	a.Log.Debug("Cluster roles completed")
 
 	for _, csv := range a.csvSet(op.Namespace, v1alpha1.CSVPhaseAny) {
-		origCSVannotations := csv.GetAnnotations()
+		origCSVannotations := a.copyOperatorGroupAnnotations(&csv.ObjectMeta)
 		a.addOperatorGroupAnnotations(&csv.ObjectMeta, op, !csv.IsCopied())
 		if reflect.DeepEqual(origCSVannotations, csv.GetAnnotations()) == false {
 			// CRDs don't support strategic merge patching, but in the future if they do this should be updated to patch
 			if _, err := a.client.OperatorsV1alpha1().ClusterServiceVersions(csv.GetNamespace()).Update(csv); err != nil {
 				a.Log.Errorf("Update for existing CSV failed: %v", err)
+			} else {
+				if _, ok := origCSVannotations[v1alpha2.OperatorGroupAnnotationKey]; ok {
+					if err := a.csvQueueSet.Requeue(csv.GetName(), csv.GetNamespace()); err != nil {
```

Comment:
Updating `ObjectMeta` on the CSV will enqueue it. I don't think you need to explicitly requeue.

Reply:
This specifically handles the case when an annotation hasn't occurred, but CSVs still need to be copied to newly created namespaces.